src/features/academic/components/shared/Accordion/CourseList/CourseList.module.scss

```
@@ -10,21 +10,30 @@
 
 .courseItem {
   display: flex;
-  justify-content: space-between;
-  align-items: center;
-  height: 26px;
+  // 과목명이 2~3줄로 길어져도 우측 메타(학기/학점/등급)는 첫 줄에 top 정렬.
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)